pynest/examples/brette_et_al_2007/coba.py

```diff
@@ -0,0 +1,116 @@
+# -*- coding: utf-8 -*-
+#
+# coba.py
+#
+# This file is part of NEST.
+#
+# Copyright (C) 2004 The NEST Initiative
+#
+# NEST is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 2 of the License, or
+# (at your option) any later version.
+#
+# NEST is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with NEST.  If not, see <http://www.gnu.org/licenses/>.
+
+"""
+Benchmark 1 of the simulator review (COBA)
+------------------------------------------
+
+This script creates a sparsely coupled network of excitatory and inhibitory
+neurons which exhibits self-sustained activity after an initial stimulus.
+Connections within and across both populations are created at random. Both
+neuron populations receive Poissonian background input. The spike output of
+500 neurons from each population are recorded. Neurons are modeled as leaky
+integrate-and-fire neurons with conductance-based synapses (exponential
+functions).
+
+The model is based on:
+
+    T.P. Vogels & L.F. Abbott
+    Signal Propagation and Logic Gating in Networks of
+    Integrate-and-Fire Neurons
+    Journal of Neuroscience, 2005, vol 25, pp 10786-10795.
+
+This is Benchmark 1 of the FACETS simulator review (Brette et al. 2007):
+- Neuron model: integrate-and-fire (i&f)
```

Comment:
In all other occurrences, too.
```suggestion
- Neuron model: integrate-and-fire (`iaf_cond_exp`)
```